sdcm/cluster.py

```diff
@@ -2175,33 +2184,38 @@ def node_setup(self, node, verbose=False, db_node_address=None, **kwargs):
         node.download_scylla_repo(self.params.get('scylla_repo'))
         if node.is_rhel_like():
             node.remoter.run('sudo yum install -y {}-tools'.format(node.scylla_pkg()))
-        else:
-            node.remoter.run('sudo apt-get update')
-            node.remoter.run('sudo apt-get install -y -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" --force-yes --allow-unauthenticated {}-tools'.format(node.scylla_pkg()))
-
-        node.wait_cs_installed(verbose=verbose)
-        if node.is_rhel_like():
             node.remoter.run('sudo yum install -y screen')
         else:
+            node.remoter.run('sudo apt-get update')
+            node.remoter.run('sudo apt-get install -y -o Dpkg::Options::="--force-confdef"'
+                             ' -o Dpkg::Options::="--force-confold" --force-yes'
+                             ' --allow-unauthenticated {}-tools'.format(node.scylla_pkg()))
             node.remoter.run('sudo apt-get install -y screen')
+
         if db_node_address is not None:
-            node.remoter.run("echo 'export DB_ADDRESS=%s' >> $HOME/.bashrc" %
-                             db_node_address)
+            node.remoter.run("echo 'export DB_ADDRESS=%s' >> $HOME/.bashrc" % db_node_address)
 
+        node.wait_cs_installed(verbose=verbose)
         cs_exporter_setup = CassandraStressExporterSetup()
         cs_exporter_setup.install(node)
-
-        if self.params.get('bench_run', default=False):
-            # go1.7 still not in repo
-            node.remoter.run('sudo yum install git -y')
-            node.remoter.run('curl -LO https://storage.googleapis.com/golang/go1.9.linux-amd64.tar.gz')
-            node.remoter.run('sudo tar -C /usr/local -xvzf go1.9.linux-amd64.tar.gz')
-            node.remoter.run("echo 'export GOPATH=$HOME/go' >> $HOME/.bashrc")
-            node.remoter.run("echo 'export PATH=$PATH:/usr/local/go/bin' >> $HOME/.bashrc")
-            node.remoter.run("source $HOME/.bashrc")
-            node.remoter.run("go get github.com/scylladb/scylla-bench")
         self.get_cassandra_stress_version(node)
 
+        # scylla-bench
+        node.remoter.run('sudo yum install git -y')
+        node.remoter.run('curl -LO https://storage.googleapis.com/golang/go1.9.2.linux-amd64.tar.gz')
+        node.remoter.run('sudo tar -C /usr/local -xvzf go1.9.2.linux-amd64.tar.gz')
+        node.remoter.run("echo 'export GOPATH=$HOME/go' >> $HOME/.bashrc")
+        node.remoter.run("echo 'export PATH=$PATH:/usr/local/go/bin' >> $HOME/.bashrc")
+        node.remoter.run("source $HOME/.bashrc")
+        node.remoter.run("go get github.com/scylladb/scylla-bench")
+
+        # gemini tool
+        node.remoter.run("cd $HOME")
```

Comment:
gemini shouldn't be installed if not needed

Reply:
As I remember it was agreed with @roydahan that we install it with no conditions

Reply:
@roydahan why do we need to install tools that are not used in the test?